libs/ardour/luabindings.cc

```diff
@@ -684,7 +684,7 @@ LuaBindings::common (lua_State* L)
 #endif
 		.addFunction ("magnitude", &Temporal::timecnt_t::magnitude)
 		.addFunction ("position", &Temporal::timecnt_t::position)
-		.addFunction ("origin", &Temporal::timecnt_t::origin)
+		.addFunction ("origin", &Temporal::timecnt_t::position)  // backwards compatibility
```

Comment:
There is no need to keep Lua API backwards compatible. For better or worse it follows libardour's API and things break on occasion.
There is also currently no script in share/scripts/*.lua that uses that API.

Reply:
That does indeed sound "worse" ;-)
Breaking a public API unnecessarily is so much against my beliefs ...